### PR TITLE
login page responsive

### PR DESCRIPTION
**Pull request by paul1124**

### CODE REVIEW

File: client/src/Components/AppStage/AppStage.scss [hunk 1]
```diff
@@ -1,5 +1,11 @@
 @import './../../variables.scss';
 
+@media only screen and (max-width: 700px) {
+  .stage {
+    display: block !important;
+  }
```
Comment (terrancexin) [inline client/src/Components/AppStage/AppStage.scss:6]: try to avoid `!important` by any means...

**Review by terrancexin — CHANGES_REQUESTED** [review]
is there an alternative approach than using `!important`?

**Review by clockwerkz — APPROVED** [review]
I left a minor note on elements breaking out when resized horizontally, but I don't think it should hold back from submission. At this point we gotta get moving.

**Review by nroode — COMMENTED** [review]
For the mobile Login view, let's remove the green 'margin' along the sides and bottom of the app.  Let's also reduce the padding a bit around the title and center the title.

I think it also might be good to adjust the colors a bit so the Login is not so distinctly placed/defined but instead blends into the background more.

**Pull request closed without merging**